src/pages/HomePage.js

```diff
@@ -20,6 +20,8 @@ import config, { FILTERS_KEY } from '../config';
 
 const FIND_BY_ID = loader('./findLooById.graphql');
 
+const SIDEBAR_BOTTOM_MARGIN = 32;
```

Comment:
Should ideally use `theme.space` but `useTheme` doesn't work directly in this component and decided not to spend ages investigating.